ethcore/src/blockchain/extras.rs

```diff
@@ -180,7 +177,8 @@ impl rlp::Decodable for BlockDetails {
 	fn decode(rlp: &rlp::Rlp) -> Result<Self, rlp::DecoderError> {
 		let use_short_version = match rlp.item_count()? {
 			4 => true,
-			6 => false,
+			// 6, cause we used to store metadata
+			5 | 6 => false,
```

Comment:
to keep database backwards compatibility

Reply:
Looks fine but I think we actually never had any `6` cases up to this point. `6` is possible after the Aura finality PR is merged. Right now we don't have any. It's also not possible to use any custom chain spec to reach the `6` case.

The only case where `6` was used is in https://github.com/paritytech/parity-ethereum/pull/8654, but we're not using that code in the short-term.